tools/tgs_scripts/InstallDeps.sh

```diff
@@ -17,11 +17,11 @@ if ! ( [ -x "$has_git" ] && [ -x "$has_curl" ] && [ -f "/usr/lib/i386-linux-gnu/
 	if ! [ -x "$has_sudo" ]; then
 		dpkg --add-architecture i386
 		apt-get update
-		apt-get install -y lib32z1 git pkg-config libssl-dev:i386 libssl-dev zlib1g-dev:i386 curl
+		apt-get install -y lib32z1 git pkg-config zlib1g-dev:i386 curl
 	else
 		sudo dpkg --add-architecture i386
 		sudo apt-get update
-		sudo apt-get install -y lib32z1 git pkg-config libssl-dev:i386 libssl-dev zlib1g-dev:i386 curl
+		sudo apt-get install -y lib32z1 git pkg-config zlib1g-dev:i386 curl
```

Comment:
Holdup, these are deps of auxtools (and therefore auxlua) as well.

Idk how, but they are.

Reply:
CC @willox @Mothblocks

Reply:
IIRC it's DMASM using an old ass spacemandmm/dreammaker version

Reply:
Does Test TGS Docker not do auxtools/auxlua stuff? Might be worth looking into in the future so we wouldn't immediately shrek ourselves on accident like we were just about to here.

I'll revert it in the meantime

Reply:
As long as it's not in `PreCompile.sh`, it doesn't